sklearn/metrics/_plot/roc_curve.py

```diff
@@ -41,10 +55,44 @@ class RocCurveDisplay(_BinaryClassifierCurveDisplayMixin):
 
         .. versionadded:: 0.24
 
+    fpr : ndarray or list of ndarray
```

Comment:
Does `fpr` also take a list of ndarray? I think that it gets converted to a list on `_deprecate_singular`. As a "user" I would be a bit confused there I think.

Reply:
That was a copy pasta! Fixed now.

These are deprecated now - they'll have the red deprecated box (via the `.. deprecated`), which I am hoping will be enough.

If the user does use it, they will get a warning and what they input will be put into a list and passed. This is all stated in the warning. I thought about adding it to the docstring here as well, but thought it wasn't necessary and would make the parameter descriptions un-necessarily long and complicated. Open to other opinions though!